src/transform_astronomy_data.py

```diff
@@ -0,0 +1,63 @@
+'''Gets JSON data from specified file, filters for relevant columns and converts into dataframe'''
+
+# pylint:disable=line-too-long
+
+import json
+import pandas as pd
+
+FILE_PATH = "data/planetary_data.json"
```

Comment:
question: should this include the path to get to the top level as well?
```suggestion
FILE_PATH = "../data/planetary_data.json"
```

Reply:
Testing now, I don't think this works when inside the src @mercuridi